templates/preview-code.html

```diff
@@ -0,0 +1,9 @@
+<!doctype html>
+<html lang=en data-inscription={{self.inscription_id}}>  <head>
+    <meta charset=utf-8>
+    <link rel=stylesheet href=/static/preview-code.css>
+    <script src=/static/preview-code.js defer type=module></script>
+  </head>
+  <body>
+  </body>
+</html>
```

Comment:
Think this needs a proper EOL at the end

Reply:
Diffs on github don't seem to show it. I see it in my IDE however.
<img width="567" alt="image" src="https://github.com/ordinals/ord/assets/42568538/5cd362ba-2f63-4b46-835d-ca9bc605fc4f">